### ISSUE TITLE
why dose my finetuned model repeat the given prompt before generating its response

### ISSUE THREAD

**Issue by starrlee356** [tatsu-lab/stanford_alpaca#320]
Million thanks to your great work! Could you help me with my problem?
After I finetune my llama model, when I prompt it with text like "Below is an instruction that describes a task, paired with an input that provides further context. Write a response that appropriately completes the request.\n ### Instruction: {my_instruct} ### Input: {my_input} ###Response:", the model does not only output the response part as I expect, it outputs the prompt above and then its response.
my dataset looks like:
![dataset](https://github.com/user-attachments/assets/b84e5fc1-0f74-46e4-8b07-992a32518d89)
my environment is:
![env1](https://github.com/user-attachments/assets/c952b576-8905-4c9e-ab1d-ec12dd317308)
![env2](https://github.com/user-attachments/assets/2c74cae7-9e2a-4f62-b7f2-e05ab47ad7f4)

**Comment by 0m364** [comment]
Try implementing a seed .. start from that seed as a baseline to test.. but . With the information provided it seems like the fine-tuning made it /think the current prompt before replying... If you don't mind me asking.. how many epochs of fine tuning??
